13/index.php

```diff
@@ -26,9 +26,9 @@
 	<ul class="navigationBar" >
 		<li class="navigationBar" onclick="previusPage()">&lt; &lt; Previous</li>
 		<li class="navigationBar" onclick="homePage()"> Home </li>
+		<li class="navigationBar" onclick="nextPage()">Next &gt; &gt;</li>
```

Comment:
The subsequent chapters are not ready. Please comment this

Reply:
returned as it was

Reply:
returned as it was